### ISSUE TITLE
Specifying multiple packages

### ISSUE THREAD

**Issue by maxov** [packson/packson#13]
There's a use for specifying multiple packages within one artifact. There's two ways to do this. First is to allow an array at the root of craft.json:

``` js
[
  { /* package */ },
  { /* package */ },
  // ...
]
```

Another option is to specify multiple packages as a key in the root package object:

``` js
{
  "subpackages": {
    "path/1": { /* package */ },
    "path/2": { /* package */ },
    // ...
   }
}
```

I'm sort of leaning to the second option because it means that we don't have to complicate the spec with two kinds of roots in craft.json. The question is if it's useful to just have a collection of 'subpackages', i.e. that specifying the root package is required.

**Comment by stephan-gh** [comment]
I would personally do it like

``` json
{
    "mymod": {
        "group": "net.minecrell"
        ...
    }
}
```

or even:

``` json
{
    "net.mygroup:mymod": {
        "name": "MyMod"
        ...
    }
}
```

This will basically allow several crafts in one package and doesn't require making the file an array if only one package is there either.

**Comment by kashike** [comment]
I'd go for this, always requiring an array:

``` json

[
    {
        "group": "org.spongepowered",
        "id": "spongeforge",
        "name": "SpongeForge"
    },
    {
        "group": "org.spongepowered",
        "id": "spongevanilla",
        "name": "SpongeVanilla"
    }
]
```

**Comment by jamierocks** [comment]
I have to say I agree with @kashike.

**Comment by maxov** [comment]
Fixed with b8c93d7150b67504
